.circleci/config.yml

```diff
@@ -54,23 +71,41 @@ workflows:
                 - develop
     jobs:
       - build:
-          production: true
+          build_config: "integreat"
       - bump_version:
           requires:
             - build
       - deliver:
           type: approval
-          production: true
+          deliver_to: "integreat"
           requires:
             - build
             - bump_version
 
+  bi-weekly-malte-delivery:
+    triggers:
+      - schedule:
+          cron: "0 5 1,15 * *" # For sanity checks for this syntax: https://crontab.guru/
+          filters:
+            branches:
+              only:
+                - develop
+    jobs:
+      - build:
+          build_config: "malte"
+      - deliver:
```

Comment:
This will not work as the type is `approval`. A step called like `await_production_approval` is required.

Reply:
```
      - await_production_approval:
          type: approval
          requires:
            - bump_version
```

Reply:
No, this is not needed. Will work anyway. No extra step required.

Reply:
Did you test this? For me it did not work when I tried it. I could click approve but the job was not executed.

Reply:
Yes, already approved one delivery.

Reply:
Edit: Alright, I see. Yeah seems like this is also the case for me, thanks for pointing it out.

Reply:
@maxammann  Should I keep the required approval or should we just deliver?

Reply:
For malte we can remove the approval, for integreat.app we should keep it.

Reply:
:+1:

Reply:
@maxammann done